src/index.html

```diff
@@ -7,6 +7,42 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__container calendar__container--month-length--29 calendar__container--start-day--sun"
+      >
+        <div class="calendar__day calendar__day--day--1"></div>
```

Comment:
don't create separate class with modifier for each day (use nth-child pseudo class instead of creating separate class for each day)